tirtha_bk/static/js/main.js

```diff
@@ -65,10 +120,24 @@ expandBtn.addEventListener("click", () => {
     }, 100);
     exitFullScreen();
   }
+>>>>>>> upstream/main
```

Comment:
This text is from a merge-conflict resolver. Shouldn't be here.

Reply:
And so on, at many places, in this file. Remove all of these and also any duplicated code.